packages/blade/package.json

```diff
@@ -30,50 +34,49 @@
     "razorpay",
     "blade"
   ],
-  "main": "./build/components/index.web.js",
   "exports": {
     "./components": {
       "react-native": {
-        "types": "./build/components/index.native.d.ts",
-        "default": "./build/components/index.native.js"
+        "types": "./components.d.ts",
+        "default": "./build/lib/native/components/index.js"
       },
       "default": {
-        "types": "./build/components/index.d.ts",
-        "development": "./build/components/index.development.web.js",
-        "production": "./build/components/index.production.web.js",
-        "default": "./build/components/index.development.web.js"
+        "types": "./components.d.ts",
+        "development": "./build/lib/web/development/components/index.js",
+        "production": "./build/lib/web/production/components/index.js",
+        "default": "./build/lib/web/production/components/index.js"
       }
     },
     "./tokens": {
       "react-native": {
-        "types": "./build/tokens/index.native.d.ts",
-        "default": "./build/tokens/index.native.js"
+        "types": "./tokens.d.ts",
+        "default": "./build/lib/native/tokens/index.js"
       },
       "default": {
-        "types": "./build/tokens/index.d.ts",
-        "development": "./build/tokens/index.development.web.js",
-        "production": "./build/tokens/index.production.web.js",
-        "default": "./build/tokens/index.development.web.js"
+        "types": "./tokens.d.ts",
+        "development": "./build/lib/web/development/tokens/index.js",
+        "production": "./build/lib/web/production/tokens/index.js",
+        "default": "./build/lib/web/production/tokens/index.js"
       }
     },
     "./utils": {
       "react-native": {
-        "types": "./build/utils/index.native.d.ts",
-        "default": "./build/utils/index.native.js"
+        "types": "./utils.d.ts",
+        "default": "./build/lib/native/utils/index.js"
       },
       "default": {
-        "types": "./build/utils/index.d.ts",
-        "development": "./build/utils/index.development.web.js",
-        "production": "./build/utils/index.production.web.js",
-        "default": "./build/utils/index.development.web.js"
+        "types": "./utils.d.ts",
+        "development": "./build/lib/web/development/utils/index.js",
+        "production": "./build/lib/web/production/utils/index.js",
+        "default": "./build/lib/web/production/utils/index.js"
       }
     }
   },
   "scripts": {
     "typecheck": "run-p types:typecheck:*",
     "test": "run-p test:*",
     "prepare": "yarn build",
-    "build": "run-s build:*",
+    "build": "./scripts/buildBlade.sh",
```

Comment:
@anuraghazra I believe it might not be compatible with Windows like before.

Reply:
removed this and used run-s